Library/Homebrew/formula_installer.rb

```diff
@@ -97,7 +97,7 @@ def install_bottle_for?(dep, build)
   end
 
   def prelude
-    verify_deps_exist unless ignore_deps?
+    verify_deps_exist unless ignore_deps? || is_a?(DependencyInstaller)
```

Comment:
What method name should we call it? How about `skip_deps_check?`? So we will have:
```ruby
class FormulaInstaller
  def skip_deps_check?
    ignore_deps?
  end
end

class DependencyInstaller < FormulaInstaller
  def skip_deps_check?
    true
  end
end
```